### PR TITLE
[Tests-Only]Bump commit IDs on ocis 2021-04-26

### PR DESCRIPTION
**Pull request by Talank**
## Description
This PR bumps to the latest web and core commit id

## Related Issue
- https://github.com/owncloud/QA/issues/662

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (phil-davis) [inline yarn.lock:1]: did you intend to commit this?

Comment (SwikritiT) [reply yarn.lock:1]: No, I might have accidentally committed it . I'll remove it

File: .gitignore [hunk 2]
```diff
@@ -1,33 +0,0 @@
-*/coverage.out
```
Comment (phil-davis) [inline .gitignore:1]: Looks like this was accidentally deleted.

**Comment by phil-davis** [comment]
Note: this PR needs to be rebased after the existing expected-failures problem issue https://github.com/owncloud/core/issues/38657 is fixed by https://github.com/owncloud/ocis/pull/1973

**Review by phil-davis — APPROVED** [review]

**Pull request merged**